composer.json

```diff
@@ -144,7 +144,7 @@
     "conflict": {
         "doctrine/dbal": "2.7.0",
         "doctrine/lexer": "1.0.0",
-        "doctrine/orm": "2.10.0 || 2.14.2 || >= 2.16.0",
+        "doctrine/orm": "2.10.0 || 2.14.2 || 2.16.0 - 2.17.2",
```

Comment:
TODO currently we require a conflict to `doctrine/lexer` `3.0` see https://github.com/sulu/sulu/pull/7256